packages/test-utils/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "tg-test-utils"
+version = "0.5.2"
```

Comment:
Use 0.5.3 here for consistency?

Reply:
True, I just did checkless move to not screw up, but I can bump it.